examples/scenes/src/svg.rs

```diff
@@ -94,16 +96,51 @@ pub fn svg_function_of<R: AsRef<str>>(
     contents: impl FnOnce() -> R + Send + 'static,
 ) -> impl FnMut(&mut Scene, &mut SceneParams) {
     fn render_svg_contents(name: &str, contents: &str) -> (Scene, Vec2) {
+        use crate::pico_svg::*;
         let start = Instant::now();
-        let svg = usvg::Tree::from_str(contents, &usvg::Options::default())
-            .unwrap_or_else(|e| panic!("failed to parse svg file {name}: {e}"));
-        eprintln!("Parsed svg {name} in {:?}", start.elapsed());
-        let start = Instant::now();
-        let mut new_scene = Scene::new();
-        vello_svg::render_tree(&mut new_scene, &svg);
-        let resolution = Vec2::new(svg.size.width() as f64, svg.size.height() as f64);
-        eprintln!("Encoded svg {name} in {:?}", start.elapsed());
-        (new_scene, resolution)
+        match PicoSvg::load(contents, 1.0) {
+            std::result::Result::Ok(PicoSvg { items, size }) => {
+                eprintln!("Parsed svg {name} in {:?}", start.elapsed());
+                let start = Instant::now();
+                let mut new_scene = Scene::new();
+                for item in items {
+                    match item {
+                        Item::Fill(fill) => {
+                            new_scene.fill(
+                                Fill::NonZero,
+                                Affine::IDENTITY,
+                                fill.color,
+                                None,
+                                &fill.path,
+                            );
+                        }
+                        Item::Stroke(stroke) => {
+                            new_scene.stroke(
+                                &Stroke::new(stroke.width),
+                                Affine::IDENTITY,
+                                stroke.color,
+                                None,
+                                &stroke.path,
+                            );
+                        }
+                    }
+                }
+                eprintln!("Encoded svg {name} in {:?}", start.elapsed());
+                (new_scene, size.to_vec2())
+            }
+            std::result::Result::Err(e) => {
+                eprintln!("Failed to load svg, with Err({e:?})");
```

Comment:
```suggestion
                eprintln!("Failed to load svg, with Err({e})");
```
Errors should implement Display, which is the better value to show for them